resources/views/settings.blade.php

```diff
@@ -3,310 +3,391 @@
     {{ __('menu.settings') }}
 @endsection
 @section('content')
-<div class="container">
-    <div class="row justify-content-center">
-        <div class="col-md-8">
-            <div class="card">
-                <div class="card-header">{{ __('settings.title-profile') }}</div>
-
-                <div class="card-body">
-                    <form enctype="multipart/form-data" method="POST" action="{{ route('settings') }}">
-                        @csrf
-
-                        <div class="form-group row">
-                            <label for="name" class="col-md-4 col-form-label text-md-right">{{ __('settings.picture') }}</label>
-
-                            <div class="col-md-6 text-center">
-                                <div class="image-box">
-                                    <img src="{{ route('account.showProfilePicture', ['username' => $user->username]) }}" style="max-width: 96px" alt="{{__('settings.picture')}}" class="pb-2" id="theProfilePicture" />
-                                </div>
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-md-8">
+                <div class="card">
+                    <div class="card-header">{{ __('settings.title-profile') }}</div>
+
+                    <div class="card-body">
+                        <form enctype="multipart/form-data" method="POST" action="{{ route('settings') }}">
+                            @csrf
+
+                            <div class="form-group row">
+                                <label for="name"
+                                       class="col-md-4 col-form-label text-md-right">{{ __('settings.picture') }}</label>
+
+                                <div class="col-md-6 text-center">
+                                    <div class="image-box">
+                                        <img
+                                            src="{{ route('account.showProfilePicture', ['username' => $user->username]) }}"
+                                            style="max-width: 96px" alt="{{__('settings.picture')}}" class="pb-2"
+                                            id="theProfilePicture"/>
+                                    </div>
 
-                                <a href="#" class="btn btn-primary" data-toggle="modal" data-target="#uploadAvatarModal">{{__('settings.upload-image')}}</a>
+                                    <a href="#" class="btn btn-primary" data-toggle="modal"
+                                       data-target="#uploadAvatarModal">{{__('settings.upload-image')}}</a>
+                                    @if($user->avatar != 'user.jpg')
+                                        <a href="#" class="btn btn-outline-danger btn-sm" data-toggle="modal"
+                                           data-target="#deleteProfilePictureModal">{{ __('settings.delete-profile-picture-btn') }}</a>
+                                    @endif
```

Comment:
Dieser State kann sich ändern, wenn ein Nutzer ein Bild hochläd (erzeugt keinen Page-Reload, alles ajax). Können wir den Link per CSS ausblenden? 
```suggestion
                                    <a href="#" class="btn btn-outline-danger btn-sm
                                       @if($user->avatar != 'user.jpg') d-none @endif"
                                       data-toggle="modal"
                                       data-target="#deleteProfilePictureModal"
                                       >{{ __('settings.delete-profile-picture-btn') }}</a>
```

Dann könnten wir den Button nach dem Upload anzeigen (`resources/js/components/settings.js` entsprechend anpassen).

Reply:
Ich schau‘s mir an